spec/support/test_setup.rb

```diff
@@ -0,0 +1,90 @@
+class TestSetup
+  # Class that instantiates standard set of test data for specs.
+
+  @@recruitment_cycles = {
+    current_cycle: -> { FactoryBot.find_or_create :recruitment_cycle },
```

Comment:
Can we get by without using `FactoryBot` here? It seems really noisy and affects reability, imo.